app/views/content/early-years-teaching-training.md

```diff
@@ -19,74 +19,56 @@ date: "2021-03-29"
 description: |-
   Discover how to get Early Years Teacher Status (EYTS) and help to educate and inspire future generations. Official guidance on early years teacher training.
 calls_to_action:
-  EY-undergraduate:
-    name: feature_table
-    arguments:
-      - Cost: "Up to £9,250"
-        Salary: "No"
-        Funding available: "Yes"
-        Length of training: "3 or 4 years"
-        Qualification: "Early years teacher status"
-      - "Undergraduate early years teacher training"
   EY-graduate:
     name: feature_table
     arguments:
-      - Cost: "£7,000"
+      - Cost: "£7000 course fees are covered by the Department for Education and are paid directly to the training provider. You don’t need to do anything to access this funding."
         Salary: "No"
-        Funding available: "Yes"
         Length of training: "Typically 1 academic year"
         Qualification: "Early years teacher status"
-      - "Graduate early years teacher training"
   EY-graduate-employment:
     name: feature_table
     arguments:
-      - Cost: "£7,000"
+      - Cost: "£7000 course fees are covered by the Department for Education and are paid directly to the training provider. You don’t need to do anything to access this funding."
         Salary: "Provided by employer"
-        Funding available: "Yes"
         Length of training: "Typically 1 academic year"
         Qualification: "Early years teacher status"
-      - "Graduate employment-based early years teacher training"
+  EY-undergraduate:
+    name: feature_table
+    arguments:
+      - Cost: "Up to £9,250"
+        Salary: "No"
+        Length of training: "3 or 4 years"
+        Qualification: "Early years teacher status"
   EY-self-funded:
     name: feature_table
     arguments:
       - Cost: "Self-funded"
```

Comment:
Do we have the actual cost for this? seems a bit strange that we don't

Reply:
Good question, I'll check with the team.

Reply:
It may vary by provider - if there is no exact figure, is there a 'Typically between x and y' statement we could add?